vendor/engines/split_accounts/app/models/split.rb

```diff
@@ -0,0 +1,19 @@
+class Split < ActiveRecord::Base
+
+  belongs_to :parent_split_account, class_name: "SplitAccount", foreign_key: :parent_split_account_id, inverse_of: :splits
+  belongs_to :subaccount, class_name: "Account", foreign_key: :subaccount_id, inverse_of: :parent_splits
+
+  scope :with_extra_penny, -> { where(extra_penny: true) }
```

Comment:
Yeah, you might be right. The fact that scopes are always chainable is nice. I don't think we use the extensions anywhere.

The one nice thing about converting over to the class methods is that it becomes an indicator of how recent it is and it's clear that it's a rails 3 scope and doesn't use the old rails 2 syntax. But that's not a very strong argument. I'm fine leaving this as `scope`.